zerver/tests/test_docs.py

```diff
@@ -537,17 +536,19 @@ def test_apps_view(self) -> None:
         self.assertEqual(result.status_code, 301)
         self.assertTrue(result["Location"].endswith("/apps/"))
 
-        with self.settings(ZILENCER_ENABLED=False):
-            result = self.client_get("/apps/")
+        with self.settings(CORPORATE_ENABLED=False):
+            with self.modify_settings(INSTALLED_APPS={"remove": "corporate"}):
+                result = self.client_get("/apps/")
```

Comment:
This isn't enough to work, for two reasons:
 - For some reason `corporate/urls.py` is still in effect, despite the app having been removed from `INSTALLED_APPS`, and its view gets called
 - Because the code in `zproject/urls.py` below is at _import-time_, it means that overriding `CORPORATE_ENABLED` does not suffice for it to go back and decide to add that path to `zerver`s paths.

I'm not sure the right path forward for this to be testable.

I'm also very confused why these tests are in `test_docs.py`.